cw_solana/lib/default_spl_tokens.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:cw_solana/spl_token.dart';
+
+class DefaultSPLTokens {
+  final List<SPLToken> initialSPLTokens = [
+    SPLToken(
+      name: 'USDT',
+      symbol: 'USDT',
+      mintAddress: 'Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB',
+      decimal: 6,
+      mint: 'usdt',
+      logoUrl:
+          "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB/logo.svg",
```

Comment:
nope, do not depend on a network image, either download the asset (I only recommend this for important tokens only and that is if we don't have the icon already) else please do like Ethereum, check if the token already is defined in `CryptoCurrency.dart` and take the image from there, else leave it as null
(that's for all of the tokens)